src/base/QXmppStanza.h

```diff
@@ -187,14 +187,15 @@ class QXMPP_EXPORT QXmppStanza
         /// \cond
         void parse(const QDomElement &element);
         void toXml(QXmlStreamWriter *writer) const;
-        /// \endcond
 
     private:
+        // ### QXmpp2: remove
         QString getConditionStr() const;
         void setConditionFromStr(const QString &cond);
 
         QString getTypeStr() const;
         void setTypeFromStr(const QString &type);
+        /// \endcond
```

Comment:
This family of functions doesn't seem to affect neither API (it's private) nor ABI (it's not virtual after all), so looks like it's safe to remove it entirely without waiting for qxmpp 2.

Reply:
Can I really just remove a private method without affecting ABI compatibility?

Reply:
https://community.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B says:
> You can...
> * remove private non-virtual functions if they are not called by any inline functions (and have never been).

Thank you, that's nice, didn't know that this was possible :)